### PR TITLE
I833 multi select

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

**Comment by Serhiy91** [comment]
@Dartv Creatable multi select field with custom value tag does not work. Need to dive deeper into react-select to fix it.

File: app/imports/ui/react/forms/components/SelectInput/index.js [hunk 1]
```diff
@@ -116,6 +164,9 @@ const enhance = compose(
     ),
     identity,
   ),
+  withProps(({ multi }) => ({
+    valueComponent: multi && MultiValue,
```
Comment (Dartv) [inline app/imports/ui/react/forms/components/SelectInput/index.js]: This is not flexible. What if we wanted to pass a custom `valueComponent`?

File: app/imports/ui/react/forms/components/SelectInput/MultiValue.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (Dartv) [inline app/imports/ui/react/forms/components/SelectInput/MultiValue.js]: Why place it inside of SelectInput folder? We may want to use this component later. Rename it to something more flexible and place it under /react/components folder. Also, `SelectInput` itself should just be a file

File: app/imports/ui/react/forms/components/SelectInput/MultiValue.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import PropTypes from 'prop-types';
+import { ButtonGroup, Button } from 'reactstrap';
+import Icon from '../../../components/Icons/Icon';
+
+const MultiValue = ({ value, onRemove }) => (
+  <ButtonGroup>
+    <Button disabled>{value.label}</Button>
+    <Button
+      className="btn-icon"
```
[the remark this answers is not in the document]
Comment (Serhiy91) [reply app/imports/ui/react/forms/components/SelectInput/MultiValue.js]: The button should be secondary. If I set "color="icon" button becomes transparent

**Comment by Serhiy91** [comment]
@Dartv fixed

**Comment by Serhiy91** [comment]
@Dartv fixed

**Pull request merged**